torch_xla/_patched_functions.py

```diff
@@ -1,7 +1,7 @@
 import inspect
 import torch
 import torch.nn as nn
-from torch._six import inf
+from torch import inf
```

Comment:
We can directly import `inf` from `torch` now.

Reply:
Hi
How can we use "from torch._six import string_classes" in torch 2.0?
Thanks in advance if can help.

Reply:
> How can we use `from torch._six import string_classes` in torch 2.0?

Hi @HungNgoCT, you can use `str` directly. Refer to [PyTorch 2.0 Release Note](https://github.com/pytorch/pytorch/releases/tag/v2.0.0):

> ## Releng
> 
> Deprecated private API `torch._six` (https://github.com/pytorch/pytorch/pull/94709)
> Removed the Python 2 and 3 compatibility library `six` and `future` and `torch._six`.
> 
> 2.0
> 
> ```python
> # from torch._six import string_classes
> str
> # from torch._six import int_classes
> int
> # from torch._six import inf, nan
> from torch import inf, nan
> # torch._six.string_classes
> str
> ```

Reply:
Thank you @XuehaiPan . Work now.